example/mix.lock

```diff
@@ -1,4 +1,4 @@
 %{
-  "artificery": {:hex, :artificery, "0.2.6", "f602909757263f7897130cbd006b0e40514a541b148d366ad65b89236b93497a", [:mix], [], "hexpm"},
-  "distillery": {:hex, :distillery, "2.0.2", "58e15f6708e355b610a48356ef170e63ec7753e35e215b107e1e87ed79c65170", [:mix], [{:artificery, "~> 0.2", [hex: :artificery, repo: "hexpm", optional: false]}], "hexpm"},
+  "artificery": {:hex, :artificery, "0.4.2", "3ded6e29e13113af52811c72f414d1e88f711410cac1b619ab3a2666bbd7efd4", [:mix], [], "hexpm"},
+  "distillery": {:git, "https://github.com/mobileoverlord/distillery.git", "7e73be84e0f7f8ac156c735bea6bc2f8f1d5636b", [branch: "release-mutable"]},
```

Comment:
It looks like the example project needs a `mix deps.update distillery` since I assume that you don't want it using your fork?